### PR TITLE
Add slowmode get subcommand

### PR DESCRIPTION
**Pull request by Atmois**
Add get as a subcommand for slowmode. Works identically to slowmode just returns the current slowmode rather than changing it.
Usage:
```
/slowmode get <channel>
{prefix}slowmode get <channel>
{prefix}slowmode g <channel>
```

### CODE REVIEW

**Comment by kzndotsh** [comment]
Beautiful

File: tux/cogs/moderation/slowmode.py [hunk 1]
```diff
@@ -76,6 +76,54 @@ async def slowmode(
             await ctx.send(f"Failed to set slowmode. Error: {error}", delete_after=30, ephemeral=True)
             logger.error(f"Failed to set slowmode. Error: {error}")
 
+    @slowmode.command(
```
[the remark this answers is not in the document]
Comment (kzndotsh) [reply tux/cogs/moderation/slowmode.py]: @Atmois this seems actually kinda nice

[the remark this answers is not in the document]
Comment (Atmois) [reply tux/cogs/moderation/slowmode.py]: > @Atmois this seems actually kinda nice

Do you mean the way the AI suggests the implementation?

[the remark this answers is not in the document]
Comment (Atmois) [reply tux/cogs/moderation/slowmode.py]: @kzndotsh as it didn't mention before

**Comment by electron271** [comment]
considering you can just hover your mouse over the clock icon on the bottom right to see slowmode even if you are immune its kind of a useless feature

**Comment by Atmois** [comment]
This doesn't exist on mobile @electron271

**Pull request merged**